apps/api/src/app/subscribers-v2/subscriber.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { ClassSerializerInterceptor, Controller, Get, Query, UseGuards, UseInterceptors } from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { UserAuthGuard, UserSession } from '@novu/application-generic';
+import { DirectionEnum, IListSubscribersRequestDto, IListSubscribersResponseDto, UserSessionData } from '@novu/shared';
+import { ApiCommonResponses } from '../shared/framework/response.decorator';
+
+import { ListSubscribersCommand } from './usecases/list-subscribers/list-subscribers.command';
+import { ListSubscribersUseCase } from './usecases/list-subscribers/list-subscribers.usecase';
+
+@Controller({ path: '/subscribers', version: '2' })
+@UseInterceptors(ClassSerializerInterceptor)
+@ApiTags('Subscribers')
+@ApiCommonResponses()
+export class SubscriberController {
+  constructor(private listSubscribersUsecase: ListSubscribersUseCase) {}
+
+  @Get('')
+  @UseGuards(UserAuthGuard)
```

Comment:
```suggestion
  @UserAuthentication()
  @ExternalApiAccessible()
```

The endpoint should be accessible with using the API keys. Also we should provide the swagger annotations for the docs and sdk generation.